### PR TITLE
Breaking the stack yaml generator into many files

### PR DESCRIPTION
**Pull request by mariari**
Creates an ASDF package for our configuration, and creates a readme

### CODE REVIEW

File: scripts/yaml-generator/README.org [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+'(string->dep-sha
+  "constraints-extras-0.3.0.2@sha256:bf6884be65958e9188ae3c9e5547abfd6d201df021bff8a4704c2c4fe1e1ae5b,1784")
+
+;; ③
+'(make-dependency-bare :name "cryptonite-0.27")
+
+;; ④
+
+'(make-dependency-github
```
Comment (Acentelles) [inline scripts/yaml-generator/README.org:89]: What's the difference between make-dependency-github and make-dependency-git?

File: scripts/yaml-generator/README.org [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  :name "phile314/tasty-silver"
+  :commit "f1f90ac3113cd445e2a7ade43ebb29f0db38ab9b")
+
+
+;; We tend to bunch of similar dependecies into a group
```
Comment (Acentelles) [inline scripts/yaml-generator/README.org:94]: dependecies ->. dependencies

File: scripts/yaml-generator/README.org [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+'(defparameter *morley-deps*
+  ;; Make the dependcy group
```
Comment (Acentelles) [inline scripts/yaml-generator/README.org:97]: dependcy -> dependency

**Review by systemf-frank — APPROVED** [review]

**Review by Acentelles — APPROVED** [review]

File: scripts/yaml-generator/README.org (hunk 1 above)
Comment (mariari) [reply scripts/yaml-generator/README.org:89]: github refers to this in stack

```
- github: phile314/tasty-silver
  commit: f1f90ac3113cd445e2a7ade43ebb29f0db38ab9b
```

git refers to this in stack

```
 - git: https://gitlab.com/morley-framework/morley.git
   commit: 53961f48d0d3fb61051fceaa6c9ed6becb7511e5
   subdirs:
    - code/morley
    - code/morley-prelude
```

Comment (mariari) [reply scripts/yaml-generator/README.org:89]: stack has a special case for github that doesn't require the URL, basically

File: scripts/yaml-generator/README.org [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+  (make-groups
+   ;; Write a header section on the dependcy
```
Comment (cwgoes) [inline scripts/yaml-generator/README.org:99]:
```suggestion
   ;; Write a header section on the dependency
```

File: scripts/yaml-generator/README.org (hunk 3 above)
Comment (cwgoes) [inline scripts/yaml-generator/README.org:97]:
```suggestion
  ;; Make the dependency group
```

File: scripts/yaml-generator/README.org (hunk 2 above)
Comment (cwgoes) [inline scripts/yaml-generator/README.org:94]:
```suggestion
;; We tend to bunch of similar dependencies into a group
```

**Review by cwgoes — APPROVED** [review]
utACK but fix typos

**Pull request merged**